enhancements/manifestlist/manifestlist-support.md

```diff
@@ -0,0 +1,467 @@
+i.e. multiple tags, while `Images` is a slice indicating that a user wants to import only a set
+of tags from one or more repositories. API reports back the import outcome in the `.status`
+property (some types below were embed to make visualization easier):
+
+```go
+type ImageStreamImportStatus struct {
+	Import *ImageStream
+	Images []ImageImportStatus
+	Repository struct {
+		Status metav1.Status
+		Images []ImageImportStatus
+		AdditionalTags []string
+	}
+}
+```
+In the end, the outcome of a single tag import or a full repository import is represented through
+one or multiple `ImageImportStatus` structs, this struct looks like this:
+
+```go
+type ImageImportStatus struct {
+	Status metav1.Status
+	Image *Image
+	Tag string
+}
+```
+
+`ImageImportStatus` struct has a reference for a single image on its `Image` field. This
+enhancement proposes the addition of another field where child images (manifests within a
+manifes list) would be listed, leaving `ImageImportStatus` struct looking like:
+
+
+```go
+type ImageImportStatus struct {
+	Status metav1.Status
+	Image *Image
+	Tag string
+	Manifests []Image
+}
+```
+
+Here the image that refers to the manifest list would be presented on `Image` property (as it is
+today for regular manifests) while all inner manifests within the manifest list would be listed
+in the `Manifests` slice.
+
+### Proposed ImageStreamImage changes
+
+ImageStreamImages are a way of referring to a specific Image within an ImageStream by its hash.
+It is useful when tagging an ImageStreamTag into another ImageStreamTag. With ImageStreamImage
+one can issue a command such as:
+
+```console
+$ oc tag imagestream0:tag0 imagestream1:tag1
+```
+
+This command would create a tag similar to this one inside "imagestream1":
+
+```yaml
+spec:
+  tags:
+  - name: tag1
+    from:
+      kind: ImageStreamImage
+      name: imagestream0@sha256:<sha256>
+```
+
+ImageStreamImage can also be used to "query details" about a specific Image, by using the
+following command a user can obtain information about a specific Image inside an ImageStream:
+
+```console
+$ oc get isimage <imagestream>@<sha of one of the tags>
+```
+
+With the introduction of the manifest list support, there won't be many details about an Image
+on its "parent" manifest (extra information belongs to child images). This enhancement proposes
+an extension to allow users to also view details for any of the child images.
+
+Both of the commands below would be valid, the first command would not return that much
+information but it works as a means of obtaining a list of all child images:
+
+```console
+$ oc get isimage <imagestream>@<sha of the parent image>
+$ oc get isimage <imagestream>@<sha of any of the child images>
+```
+
+The output of the `oc describe isimage` command line will need to be changed to display
+information present in the DockerImageManifests property of the parent Image.
```

Comment:
/cc @soltysh 

fyi, we want to update `oc describe` for some image related types

Reply:
No objections from my end.